internal/bminventory/inventory.go

```diff
@@ -6158,6 +6174,178 @@ func (b *bareMetalInventory) V2UpdateHostIgnitionInternal(ctx context.Context, p
 	return &h.Host, nil
 }
 
+// getOVERegistriesConf returns the registry configuration for OVE (OpenShift Virtualization Environment) images.
+// This configuration maps all required registries to a local registry that will be available in the disconnected environment.
+func getOVERegistriesConf() string {
+	return `[[registry]]
+location = "quay.io/openshift-release-dev/ocp-release"
```

Comment:
Since this is just a constant it should probably be a constant somewhere rather than in this function.